### ISSUE TITLE
issue affected list

### ISSUE THREAD

**Issue by avelino** [cstate/cstate#25]
Within an **issue** is declared which **systems** has been **affected**, it is not listed within the page and home which system the issue affected.
I think it's important to keep this public data even for historical

Appropriate location  `layouts/issues/issue.html`:
```
<small class="date">
    {{ .Date.Format "January 02, 2006 at 3:04 PM" }}
</small>
{{ range .Params.Affected }}
    <kbd>{{ . }}</kbd>
{{ end }}
```

**Comment by avelino** [comment]
like tags idea

**Comment by mistermantas** [comment]
Closing this, since it’ll be implemented quite nicely in v3.0 —

![image](https://user-images.githubusercontent.com/11616378/45253997-990ccb00-b378-11e8-9ab3-1e0e7c79624b.png)
![image](https://user-images.githubusercontent.com/11616378/45254003-9f9b4280-b378-11e8-910b-efd2f7628640.png)
![image](https://user-images.githubusercontent.com/11616378/45254009-a9bd4100-b378-11e8-9799-e3a11d72632d.png)

Affected systems are linked pretty much everywhere I can think of, even on the homepage summary.
